Tests/Runtime/MobiledgeX_RuntimeTests.cs

```diff
@@ -213,32 +212,38 @@ public async Task<bool> FindCloudletHelper()
             return foundCloudlet;
         }
 
-        public async Task<bool> VerifyLocationHelper()
+        public async Task<string> GetUrlHelper(string orgName, string appName, string appVers, string proto)
         {
-            bool verified = await integration.VerifyLocation();
-            await Task.Delay(TimeSpan.FromMilliseconds(200));
-
-            return verified;
-        }
 
-        public async Task<string> RegisterAndFindCloudletHelper(string orgName, string appName, string appVers)
-        {
-
-            FindCloudletReply reply = await integration.matchingEngine.RegisterAndFindCloudlet(orgName, appName, appVers, testLocation, MatchingEngine.wifiCarrier);
+            FindCloudletReply reply = await integration.matchingEngine.RegisterAndFindCloudlet(orgName, appName, appVers, testLocation, "");
             await Task.Delay(TimeSpan.FromMilliseconds(200));
-            string uri = reply.fqdn;
-            return uri;
+            DistributedMatchEngine.AppPort appPort;
+
+            switch (proto)
+			{
+                case "http":
+                     appPort = integration.GetAppPort(DistributedMatchEngine.LProto.L_PROTO_HTTP);
```

Comment:
I think we use L_PROTO_TCP for this. Backend doesn't support L_PROTO_HTTP

Reply:
Same for the default

Reply:
We had a discussion about this and  GetAppPort(http Proto ) frankling fixed so it returns TCP ports

Reply:
^ TCP AppPort, same return type, right?

Reply:
Yes
<img width="458" alt="Screen Shot 2020-07-02 at 4 21 43 PM" src="https://user-images.githubusercontent.com/24863504/86418328-a2210780-bc84-11ea-9cda-59d4dfe3bcf3.png">
<img width="320" alt="Screen Shot 2020-07-02 at 4 20 51 PM" src="https://user-images.githubusercontent.com/24863504/86418334-a6e5bb80-bc84-11ea-92a9-e47b3247e480.png">